src/main/java/net/dzikoysk/funnyguilds/command/admin/MoveCommand.java

```diff
@@ -33,27 +36,12 @@
         acceptsExceeded = true,
         playerOnly = true
     )
-    public void execute(CommandSender sender, String[] args) {
-        MessageConfiguration messages = FunnyGuilds.getInstance().getMessageConfiguration();
-        PluginConfiguration config = FunnyGuilds.getInstance().getPluginConfiguration();
-        Player player = (Player) sender;
-
-        if (!config.regionsEnabled) {
-            player.sendMessage(messages.regionsDisabled);
-            return;
-        }
-        
-        if (args.length < 1) {
-            player.sendMessage(messages.generalNoTagGiven);
-            return;
-        }
-
-        Guild guild = GuildUtils.getByTag(args[0]);
+    public void execute(MessageConfiguration messages, PluginConfiguration config, CommandSender sender, String[] args) {
+        when (!config.regionsEnabled, messages.regionsDisabled);
+        when (args.length < 1, messages.generalNoTagGiven);
 
-        if (guild == null) {
-            player.sendMessage(messages.generalNoGuildFound);
-            return;
-        }
+        Player player = (Player) sender;
```

Comment:
Command is marked as `playerOnly`, so you can replace CommandSender with Player in method parameters